mlir/include/mlir/Interfaces/LoopLikeInterface.td

```diff
@@ -218,6 +218,19 @@ def LoopLikeOpInterface : OpInterface<"LoopLikeOpInterface"> {
         return ::mlir::failure();
       }]
     >,
+    InterfaceMethod<[{
+        TODO
+      }],
```

Comment:
Doc?

Reply:
Oops! Sorry forgot. Added documentation.

Reply:
great, thanks for factoring out this functionality.